package.json

```diff
@@ -93,6 +93,7 @@
     "jest-serializer-vue": "^2.0.2",
     "json-loader": "^0.5.4",
     "launch-editor-middleware": "^2.2.1",
+    "lerna": "^2.11.0",
```

Comment:
Where is this used?

Reply:
It allows us to create separate npm packages for Hashi, since HTML5 apps supporting Kolibri will need to depend on it.